packages/esm-patient-chart-app/src/actions-buttons/start-visit.component.tsx

```diff
@@ -10,11 +10,15 @@ interface StartVisitOverflowMenuItemProps {
 const StartVisitOverflowMenuItem: React.FC<StartVisitOverflowMenuItemProps> = ({ patientUuid }) => {
   const { t } = useTranslation();
   const { currentVisit } = useVisit(patientUuid);
+  const { patient } = usePatient(patientUuid);
   const handleClick = React.useCallback(() => launchPatientWorkspace('start-visit-workspace-form'), []);
   const { startVisitLabel } = useConfig();
+  
+  const isDeceased = Boolean(patient?.deceasedDateTime);
+  const deceased = (!isDeceased && currentVisit);
```

Comment:
The logic here is wrong. However, I wouldn't create this confusingly named `deceased` variable at all. Instead, in the `return` use:

```ts
!currentVisit && !isDeceased && /* rest of the component */
```

This makes the logic much clearer.